ts/src/profilers/heap-profiler.ts

```diff
@@ -41,14 +42,17 @@ export function profile(): perftools.profiles.IProfile {
   // TODO: remove any once type definition is updated to include external.
   // tslint:disable-next-line: no-any
   const {external}: {external: number} = process.memoryUsage() as any;
-  const externalNode: AllocationProfileNode = {
-    name: '(external)',
-    scriptName: '',
-    children: [],
-    allocations: [{sizeBytes: external, count: 1}],
-  };
-  result.children.push(externalNode);
-  return serializeHeapProfile(result, startTimeNanos, heapIntervalBytes);
+  if (external > 0) {
```

Comment:
This seems like an independent change?

Reply:
Sorry. I made this change to simplify testing, so I wouldn't have to add an "(external)" function and location to the profiles I added. 
Should I undo this?